### PR TITLE
docs: cleanup README; add link to chemrxiv

### PR DESCRIPTION
**Pull request by byooooo**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Actions (for code changes)

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kjappelbaum** [comment]
:D we had https://github.com/kjappelbaum/pyepal/pull/84

**Comment by kjappelbaum** [comment]
I will merge yours but we should also change it in the Sphinx

File: README.md [hunk 1]
```diff
@@ -59,19 +59,19 @@ gpr_objective_1 = GaussianProcessRegressor(RBF())
 # the design space (X, in ML terms "feature matrix") and the number of objectives
 palsklearn_instance = PALSklearn(X, [gpr_objective_0, gpr_objective_1], 2)
 
-# the next step is to provide some initial measurements,
-# you can do this with the update_train_set function, which you one
-# also uses through the active learning process to update the training set.
-# For this, you provide a numpy array of indices in your design space
+# the next step is to provide some initial measurements.
+# You can do this with the update_train_set function, which you
+# cann use throughout the active learning process to update the training set.
```
Comment (kjappelbaum) [inline README.md:64]: can

**Comment by byooooo** [comment]
haha oops! yea sorry. just saw your other PR!

File: README.md [hunk 2]
```diff
@@ -117,12 +117,12 @@ class PALSklearn(PALBase):
...
 ```
 
-For scheduling for the hyperparameter optimization we have some predefined schedules in the `pyepal.pal.schedules` module.
+For scheduling of the hyperparameter optimization, we have some predefined schedules in the `pyepal.pal.schedules` module.
 
 ### Test the algorithms
 
-If the full design space is known, you can use a while loop to fully explore the space.
-For the theoretical guarantees to hold, you'll need to sample until all uncertainties are below epsilon. In practice, it is usually enough to require as termination criterion that there are no unclassified samples left. For this you could use the following snippet
+If the full design space is known, you can use a while loop to fully explore the space with ePAL.
```
Comment (kjappelbaum) [inline README.md:124]: shall we write PyePAL instead of ePAL?

File: README.md [hunk 3]
```diff
@@ -117,12 +117,12 @@ class PALSklearn(PALBase):
...
+For the theoretical guarantees of ePAL to hold, you'll need to sample until all uncertainties are below epsilon. In practice, it is usually enough to require as a termination criterion that there are no unclassified samples left. For this you can use the following snippet
```
Comment (kjappelbaum) [inline README.md:125]: shall we write PyePAL instead of ePAL?

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
the only thing that probably should be fixed is the "cann" -> "can". Thanks 👍

File: README.md (hunk 2 above)
Comment (byooooo) [reply README.md:124]: yep, i think that's better

File: README.md (hunk 3 above)
Comment (byooooo) [reply README.md:125]: yep

File: README.md (hunk 1 above)
Comment (byooooo) [reply README.md:64]: let's correct before merge

**Review by byooooo — COMMENTED** [review]
include corrections before merge

**Review by byooooo — COMMENTED** [review]
committed corrections to PR

**Review by kjappelbaum — APPROVED** [review]
lgtm, thanks

**Pull request merged**